client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -10,12 +11,15 @@ var page = require( 'page' ),
  */
 var observe = require( 'lib/mixins/data-observe' ),
 	EmptyContent = require( 'components/empty-content' ),
+	fetchSitePlans = require( 'state/sites/plans/actions' ).fetchSitePlans,
+	FreeTrialNotice = require( './free-trial-notice' ),
+	getPlansBySiteId = require( 'state/sites/plans/selectors' ).getPlansBySiteId,
 	SidebarNavigation = require( 'my-sites/sidebar-navigation' ),
 	RegisterDomainStep = require( 'components/domains/register-domain-step' ),
 	UpgradesNavigation = require( 'my-sites/upgrades/navigation' ),
 	Main = require( 'components/main' );
 
-module.exports = React.createClass( {
+var DomainSearch = React.createClass( {
 	displayName: 'DomainSearch',
```

Comment:
Do we still need to define `displayName` since we assign the component to a variable?